scripts/binary_release.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -eo pipefail
```

Comment:
Set `-x` too, so that CI prints the commands being executed and it is actually possible to debug things when they explode (in the worst case – spuriously.)